contrib/ruby_event_store-outbox/spec/tempo_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+module RubyEventStore
+  module Outbox
+    RSpec.describe Tempo do
+      specify "starts with 1" do
+        tempo = Tempo.new(100)
+
+        expect(tempo.batch_size).to eq(1)
+      end
+
+      specify "each batch size is bigger" do
+        tempo = Tempo.new(100)
+        first_value = tempo.batch_size
+        second_value = tempo.batch_size
+
+        expect(second_value).to be > first_value
+      end
+
+      specify "maximum batch size cap is respected" do
+        max_batch_size = 100
+        tempo = Tempo.new(max_batch_size)
+        100.times { tempo.batch_size }
+
+        value101 = tempo.batch_size
+        expect(value101).to eq(max_batch_size)
```

Comment:
`maximum batch size cap is respected` but what is tested here is only the value is the same at some point - no test if this really the value provided as maximum batch size

Reply:
hmm, way I see it, this test ensures that the maximum batch size is reached "at some point", and the second test ensures that the value stabilizes "and doesn't go any higher"

Reply:
...but, it could be that the maximum batch size value is reached but it stabilizes at some other (higher) point. Right. I will improve these